@wader build failure with log

```
Building subPackage ./dev
text/template/parse
text/template
github.com/wader/fq/dev
# github.com/wader/fq/dev
dev/tmpl.go:36:6: main redeclared in this block
        /private/tmp/nix-build-fq-0.0.6.drv-0/source/dev/fuzzbytes.go:12:6: previous declaration
```

--- a/pkgs/development/tools/fq/default.nix
+++ b/pkgs/development/tools/fq/default.nix
@@ -7,22 +7,24 @@
 
 buildGoModule rec {
   pname = "fq";
-  version = "0.0.4";
+  version = "0.0.6";
 
   src = fetchFromGitHub {
     owner = "wader";
     repo = "fq";
     rev = "v${version}";
-    sha256 = "sha256-mnb/9zcFSeBasGPYSGWoBhnldljGW3VK070zTO8M/uk=";
+    sha256 = "sha256-/9TBnhFGYNOcCsQKUF0uuJEgnF+qRGly/5z1s3sYhqY=";
   };
 
-  vendorSha256 = "sha256-KPIO/ZuiwxlnjGLaEuClkDsJnx/fwW0jPDBc7aTT68A=";
+  vendorSha256 = "sha256-zvtYyNJO4QoTes3vf6CFa3dYMJqkp0PG9pnOk+aO97Y=";
 
   ldflags = [
     "-s"
     "-w"
     "-X main.version=${version}"
   ];
+
+  subPackages = [ "." ];
 
   passthru.tests = testVersion { package = fq; };
 

